src/app/find/mapPage/filters/filters.css

```diff
@@ -29,3 +29,13 @@ input#hivNutrition-checkbox {
 	margin-left: 32px;
 	color: #333333;
 }
+
+.filterSubmitBtn {
+    position: fixed;
+    padding: 16px;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    z-index: 2;
+    background-color: rgba(255, 255, 255, 0.82);
```

Comment:
This isn't one of the greys we already have in colors.js (or very similar to one of them)?